src/neptune_pytorch/impl/__init__.py

```diff
@@ -207,7 +217,23 @@ def __del__(self):
             self._vis_hook_handler.remove()
 
 
-def safe_upload(run, name, model):
+def safe_upload_visualization(run, name, file_name):
```

Comment:
One additional idea: would it be possible to add type hints to the function and method params?

Reply:
Can be also a separate PR, but at some point would be nice to have them - it's then much easier to maintain :smile: